src/app/pages/ManageAssets/ManageTokens.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { memo, useMemo } from 'react';
+
+import { useAreAssetsLoading } from 'app/store/assets/selectors';
+import { useTokensMetadataLoadingSelector } from 'app/store/tokens-metadata/selectors';
+import { TEMPLE_TOKEN_SLUG } from 'lib/assets';
+import { useAllAvailableTokens } from 'lib/assets/hooks';
+import { useFilteredAssetsSlugs } from 'lib/assets/use-filtered';
+import { isSearchStringApplicable } from 'lib/utils/search-items';
+
+import { AssetsPlaceholder } from './AssetsPlaceholder';
+import { ListItem } from './ListItem';
+import { WRAPPER_CLASSNAME, ManageAssetsContent } from './ManageAssetsContent';
+import { ManageAssetsCommonProps } from './utils';
+
+export const ManageTokens = memo<ManageAssetsCommonProps>(
+  ({ chainId, publicKeyHash, removeItem, toggleTokenStatus }) => {
+    const tokens = useAllAvailableTokens(publicKeyHash, chainId);
+
+    const managebleSlugs = useMemo(
+      () => tokens.reduce<string[]>((acc, { slug }) => (slug === TEMPLE_TOKEN_SLUG ? acc : acc.concat(slug)), []),
+      [tokens]
+    );
+    const slugs = managebleSlugs;
+
+    const assetsAreLoading = useAreAssetsLoading('tokens');
+    const metadatasLoading = useTokensMetadataLoadingSelector();
+    const isLoading = assetsAreLoading || metadatasLoading;
+
+    const { filteredAssets, searchValue, setSearchValue } = useFilteredAssetsSlugs(slugs, false);
+
+    const isInSearchMode = isSearchStringApplicable(searchValue);
+
+    return (
+      <ManageAssetsContent ofCollectibles={false} searchValue={searchValue} setSearchValue={setSearchValue}>
+        {filteredAssets.length === 0 ? (
+          <AssetsPlaceholder isInSearchMode={isInSearchMode} isLoading={isLoading} />
+        ) : (
+          <div className={WRAPPER_CLASSNAME}>
```

Comment:
This fragment is similar to https://github.com/madfish-solutions/templewallet-extension/pull/1018/files#diff-48f226c0f3cd307fdcba911dd97ea7b664b6a23a8b5bbd0b14baaf9db036517eR34

Reply:
I also meant that the children are created according to similar rules